app/views/docs/command-line.phtml

```diff
@@ -118,219 +72,44 @@ brew install --HEAD appwrite</code></pre>
-        <td>init</td>
-        <td>The init command helps you initialize your Appwrite project, functions and collections</td>
-    </tr>
-    <tr>
-        <td>deploy</td>
-        <td>The deploy command provides a convenient wrapper for deploying your functions and collections</td>
-    </tr>
-    <tr>
-        <td>logout</td>
-        <td>The logout command allows you to logout from the CLI</td>
-    </tr>
-    <tr>
-        <td>projects</td>
-        <td>The projects command allows you to view, create and manage your Appwrite projects</td>
-    </tr>
-    <tr>
-        <td>storage</td>
-        <td>The storage command allows you to manage your project files</td>
-    </tr>
-    <tr>
-        <td>teams</td>
-        <td>The teams command allows you to group users of your project and to enable them to share read and write access to your project's resources</td>
-    </tr>
-    <tr>
-        <td>users</td>
-        <td>The users command allows you to manage your project users</td>
-    </tr>
-    <tr>
-        <td>client</td>
-        <td>The client command allows you to configure your CLI</td>
-    </tr>
-    <tr>
-        <td>account</td>
-        <td>The account command allows you to authenticate and manage a user account</td>
-    </tr>
-    <tr>
-        <td>avatars</td>
-        <td>The avatars command aims to help you complete everyday tasks related to your app image, icons, and avatars</td>
-    </tr>
-    <tr>
-        <td>functions</td>
-        <td>The functions command allows you view, create and manage your Cloud Functions</td>
-    </tr>
-    <tr>
-        <td>databases</td>
-        <td>The databases command allows you to create structured collections of documents, query and filter lists of documents</td>
-    </tr>
-    <tr>
-        <td>health</td>
-        <td>The health command allows you to both validate and monitor your Appwrite server's health</td>
-    </tr>
-    <tr>
-        <td>locale</td>
-        <td>The locale command allows you to customize your app based on your users' location</td>
-    </tr>
-    </tbody>
-</table>
-
-<p>Here are a few sample commands to get you started</p>
-
-<p><b>Create User</b></p>
-
-<p>To create a new user in your project, you can use the `create` command. To successfully create a user, make sure your API key is granted with the scope "users.write".</p>
+<h2><a href="/docs/command-line#next-steps" id="next-steps">Next Steps</a></h2>
+<p>Besides utility commands, the CLI can be used to execute commands like a Server SDK.</p>
+<p>
+<a href="/docs/command-line-commands"><i class="icon-angle-circled-right"></i>Find a full list of commands</a>
+</p>
+<p>You can use the CLI to create and deploy functions and collections. Deploy commands allow you to configure your Appwrite project programmatically and replicate functions and collection schemas across Appwrite projects.</p>
+<p>
+<a href="/docs/command-line-deployment"><i class="icon-angle-circled-right"></i>Learn more about deployment</a>
```

Comment:
Should the order be swapped here?

Reply:
Let's swap to match nav

Reply:
The link looks like it's in the wrong place:

![image](https://user-images.githubusercontent.com/1477010/216732717-43e74810-0122-43f6-8b7e-df02e0f35016.png)

Reply:
> The link looks like it's in the wrong place:
> 
> ![image](https://user-images.githubusercontent.com/1477010/216732717-43e74810-0122-43f6-8b7e-df02e0f35016.png)

Ughhhhhh Yes nice catch [😂](https://emojipedia.org/face-with-tears-of-joy/)[😂](https://emojipedia.org/face-with-tears-of-joy/)